pkg/otlp/model/translator/metrics_translator.go

```diff
@@ -437,6 +438,12 @@ func (t *Translator) source(m pcommon.Map) (source.Source, error) {
 
 // MapMetrics maps OTLP metrics into the DataDog format
 func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consumer Consumer) error {
+	var runtimeMetrics map[string]map[string]string
+	_, err := toml.DecodeFile("pkg/otlp/model/translator/runtime_metrics_mappings.toml", &runtimeMetrics)
```

Comment:
Why not a basic Go struct? Why introduce toml and other files? What advantage do you get from that?

Reply:
Addressed this in RFC and talked with Dinesh, we're going to use a basic Go struct instead. The main advantage of using TOML is for extensibility with OTLP intake, which we can address in the intake RFC.